electionleaflets/apps/leaflets/models.py

```diff
@@ -1,14 +1,22 @@
+import os.path
 import re
 from io import BytesIO
+from pathlib import Path
 
 import piexif
+from django.core.files import File
+from django.core.files.storage import default_storage
+from slugify import slugify
+
 from constituencies.models import Constituency
 from core.helpers import geocode
```

Comment:
Here we are importing the `geocode` function, but this function is not called anywhere in this file and this is the only place where we are importing it.
I think we can actually completely bin the geocode function, which in turn completely removes the dependency on mapit and any related code.
Unused imports is something a linter could help us catch. More on this later..